test/Card-test.js

```diff
@@ -5,27 +5,27 @@ const Card = require('../src/Card');
 
 describe('Card', function() {
 
-  it.skip('should be a function', function() {
+  it('should be a function', function() {
     const card = new Card();
     expect(Card).to.be.a('function');
   });
 
-  it.skip('should be an instance of Card', function() {
+  it('should be an instance of Card', function() {
     const card = new Card();
     expect(card).to.be.an.instanceof(Card);
   }); 
 
```

Comment:
There was no test to check if a new instance of card stores an id, this should definitely be tested for once I get to TDD.